torchmdnet/models/tensornet.py

```diff
@@ -265,10 +269,13 @@ def forward(
         x = torch.cat((tensor_norm(I), tensor_norm(A), tensor_norm(S)), dim=-1)
         x = self.out_norm(x)
         x = self.act(self.linear((x)))
+        v = skewtensor_to_vector(A)
+        v = v.transpose(1, 2)
         # # Remove the extra atom
         if self.static_shapes:
             x = x[:-1]
-        return x, None, z, pos, batch
+            v = v[:-1]
+        return x, v, z, pos, batch
```

Comment:
I am worried about the consequences of returning a vector but not using it down the line. On one hand this can mess with autograd and on the other it is wasted computation when the vector features are not used by the output model.

Reply:
Just added it.